packages/backend/src/modules/metric/metric.service.ts

```diff
@@ -209,4 +254,19 @@ export default class MetricService {
       return {};
     }
   }
+
+  private getBoundries(startDate: moment.Moment, endDate: moment.Moment, scale: 'day' | 'hour' | 'week' | 'month'): Date[] {
```

Comment:
Do we really need MomentJS here? It is slow, consider replacing it with native Date or with `date-fns`.